Makefile

```diff
@@ -1,14 +1,14 @@
 # Build the caddy binary and copy it into the proxy subdirectory
 caddy-v2-with-forwardproxy: Dockerfile proxy/Caddyfile
 	docker compose build
-	docker compose up -d 
-	docker compose cp caddy:/usr/bin/caddy proxy/caddy
+	docker compose up -d
+	- docker compose cp caddy:/usr/bin/caddy proxy/caddy
 	docker compose down
 
 validate:
 	echo "test.gov" > allow.acl
 	echo "test.com" > deny.acl
-	sed -i 's/tls cert.pem key.pem/# tls cert.pem key.pem/g' proxy/Caddyfile
+	sed -i.bak 's/tls cert.pem key.pem/# tls cert.pem key.pem/g' proxy/Caddyfile && rm proxy/Caddyfile.bak
 	PORT=9999 PROXY_USERNAME=admin PROXY_PASSWORD=pass PROXY_PORTS=443 ./proxy/caddy validate --config proxy/Caddyfile
-	sed -i 's/# tls cert.pem key.pem/tls cert.pem key.pem/g' proxy/Caddyfile
-	rm allow.acl deny.acl
+	sed -i.bak 's/# tls cert.pem key.pem/tls cert.pem key.pem/g' proxy/Caddyfile
+	rm proxy/Caddyfile.bak allow.acl deny.acl
```

Comment:
Same here... Why create a .bak file if we have no intention of leaving it around?

Reply:
because `sed -i 's...` doesn't work on macos (-i requires an argument) and the internet says that while `sed -i ''` will work on macos it will break on linux. The only fully cross-platform way is to create the backup file and then remove it

Reply:
Ahhhh, so that's the M1-specific update you made... I was wondering.

Do you want to leave a comment to that effect, with a reference? That will stop ignorant folks like me from removing it in future updates.

Reply:
would it make more sense to just install gnu-sed with homebrew?

https://daoyuan.li/a-normal-sed-on-mac/